package.json

```diff
@@ -23,7 +23,7 @@
   "license": "Apache-2.0",
   "bugs": "https://github.com/immutable/imx-core-sdk/issues",
   "engines": {
-    "node": ">=14.0.0 <17.0.0"
+    "node": ">=14.0.0 <19.0.0"
```

Comment:
Is there a reason to have the upperbound limit?

Reply:
Yeah it can be, wasn't sure if we strictly wanted nothing above v18.x as v19 is the "LTS"?

Reply:
Sure thing, I do agree, I wasn't sure if this was already considered in the ticket as the A/C was to allow for Node 18 specifically. Happy to remove the upper bound and we can revisit if there's any issues with later versions

Reply:
Yep please do